src/avlizer.erl

```diff
@@ -25,13 +25,25 @@
         ]).
 
 start(_StartType, _StartArgs) ->
-  case osenv("AVLIZER_CONFLUENT_SCHEMAREGISTRY_URL", false) of
-    false -> ok;
-    URL ->
-      Vars0 = application:get_env(?APPLICATION, avlizer_confluent, #{}),
-      Vars = Vars0#{schema_registry_url => URL},
-      application:set_env(?APPLICATION, avlizer_confluent, Vars)
+  SchemaRegistryURL = osenv("AVLIZER_CONFLUENT_SCHEMAREGISTRY_URL", false),
+  SchemaRegistrySASLMechanism = osenv("AVLIZER_CONFLUENT_SCHEMAREGISTRY_SASL_MECHANISM", false),
+  SchemaRegistrySASLUsername = osenv("AVLIZER_CONFLUENT_SCHEMAREGISTRY_SASL_USERNAME", false),
+  SchemaRegistrySASLPassword = osenv("AVLIZER_CONFLUENT_SCHEMAREGISTRY_SASL_PASSWORD", false),
```

Comment:
It's a good practice to wrap passwords in a closure, so they don't end up in the crash reports:

```
 SchemaRegistrySASLPassword0 = osenv("AVLIZER_CONFLUENT_SCHEMAREGISTRY_SASL_PASSWORD", false),
 SchemaRegistrySASLPassword = fun() ->SchemaRegistrySASLPassword0 end,
 ...
```